components/CreateGiftCardsForm.js

```diff
@@ -393,6 +406,7 @@ class CreateGiftCardsForm extends Component {
     return (
       <Container display="flex" flexDirection="column" width={1} justifyContent="center">
         <Flex justifyContent="center" mt={3} mb={4} alignItems="center">
+          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
Why is this needed, since the `input` defines the ID just below it seems that this should be considered as valid by the rule?

Reply:
@Betree : Yes that's true, I think this is a false positive; somehow the plugin triggers the rule in this case. This is why I thought of tagging  @brendanmorrell in my previous post. Let me open a issue on the `styled-components-a11y` repo as well; https://github.com/brendanmorrell/eslint-plugin-styled-components-a11y/issues/18

Reply:
https://github.com/brendanmorrell/eslint-plugin-styled-components-a11y/issues/18#issuecomment-837229005